themes/classic/_dev/js/checkout.js

```diff
@@ -51,9 +51,21 @@ function setUpCheckout() {
   });
 }
 
+function toggleImage() {
+  $('.card-block .cart-summary-products p a').on('click', function (icon) {
```

Comment:
```
.card-block .cart-summary-products p a
```

What does this selector match to? Please store it in a variable or add a comment so that we can know.

Reply:
I added a comment